README.md

```diff
@@ -44,6 +44,9 @@ Run your [Spark on AWS EMR](http://docs.aws.amazon.com/emr/latest/ReleaseGuide/e
 
   //(optional) Total number of instances, including master node. The default value is 1.
   sparkInstanceCount := 2
+
+   //(optional) 
+    sparkSubmitConfs := Map("spark.executor.cores"->"10","spark.executor.instances" -> "2")
```

Comment:
Can we move this line to the "Other available settings" section and remove the preceding whitespaces?

Reply:
Done.